Formula/virtualfish.rb

```diff
@@ -0,0 +1,91 @@
+class Virtualfish < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python virtual environment manager for the fish shell"
+  homepage "https://virtualfish.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/54/2f/a7800ae09a689843b62b3eb423d01a90175be5795b66ac675f6f45349ca9/virtualfish-2.5.5.tar.gz"
+  sha256 "6b654995f151af8fca62646d49a62b5bf646514250f1461df6d42147995a0db2"
+  license "MIT"
+  head "https://github.com/justinmayer/virtualfish.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(%r{href=.*?/packages.*?/virtualfish[._-]v?(\d+(?:\.\d+)*(?:[a-z]\d+)?)\.t}i)
+  end
```

Comment:
```suggestion
```
We can remove this. `livecheck` already knows how to handle `pythonhosted` URLs.